local_planner/launch/example.launch

```diff
@@ -1,36 +1,28 @@
 <launch>
-
-
-    <arg name="mavros_transformation" default="0" />
-    
     <param name="use_sim_time" value="false" />
 
-    <node pkg="tf" type="static_transform_publisher" name="tf_90_deg"
-          args="0 0 0 $(arg mavros_transformation) 0 0 world local_origin 10"/>
+    <!-- Transformation between the fcu and camera frame. It needs to be changed
+    according to the specific setup. Please measure the dispacement between the
+    camera and the flight controller in ENU body frame and update the x, y, z,
+    yaw, pitch, and roll in meters-->
     <node pkg="tf" type="static_transform_publisher" name="tf_front_camera"
           args="0.30 0.32 -0.11 0 0 0 fcu front_camera_link 10"/>
-                
+
     <arg name="headless" default="false"/>
     <arg name="ns" default="/"/>
     <arg name="build" default="posix_sitl_default"/>
-    <arg name="fcu_url" default="udp://:14540@localhost:14557"/>
-    <arg name="gcs_url" default="" />   <!-- GCS link is provided by SITL -->
+
+    <!-- Change the serial or udp port to match your setup-->
```

Comment:
And if you have something that should be changed I'd separate it from everything by a line below too. Also be very clear about the field being changed.
Maybe like:
```
<!-- Change fcu_url arg default attribute to the port used in your setup (serial or udp) -->
    <arg name="fcu_url" default="/dev/ttyACM0:57600"/>

    <arg name="gcs_url" default="" />
```